### DISCUSSION TITLE
讨论：如何增加文本点击事件

### DISCUSSION THREAD

**Discussion by huangzhang-ops** [wangeditor-next/wangEditor-next#304] [Q&A] [answered]
## 功能描述

如何给文本增加点击事件

## 提炼几个功能点

1、选中文本的文本点击之后可以做自己的逻辑处理

## 原型图

*涉及到 UI 改动的功能，请一定提供原型图。原型图能表明功能即可，不要求规范和美观*

## 可参考的案例

*是否已有可参考的案例（如其他编辑器），有的话请给出链接*

**Comment by cycleccc** [comment]
选中文本后点击该选中文本做些类似什么的逻辑处理呢，有更具体的描述或是可以参考的例子吗

**Comment by huangzhang-ops** [comment]
比如可以设置某些文本可以点击，类似超链接，但不是只跳转，可以弹窗等其他定制化的内容。

**Comment by cycleccc** [comment, marked as answer]
这个通过自定义元素可以解决。
具体看文档 https://cycleccc.github.io/docs/guide/development#%E5%AE%9A%E4%B9%89%E6%96%B0%E5%85%83%E7%B4%A0
以及相关实现 https://cycleccc.github.io/docs/guide/plugins
更多的github上也有很多开源的自定义插件可以对照仿写
